docs/tutorial/connecting.rst

```diff
@@ -72,12 +72,15 @@ Instead of passing a string, you could equally pass an adapter object. ::
     adapter = VISAAdapter("GPIB::4")
     sourcemeter = Keithely2400(adapter)
 
-To instead use a Prologix GPIB device connected on :code:`/dev/ttyUSB0` (proper permissions are needed in Linux, see :class:`PrologixAdapter <pymeasure.adapters.PrologixAdapter>`), the adapter is constructed in a similar way. Unlike the VISA adapter which is specific to each instrument, the Prologix adapter can be shared by many instruments. Therefore, they are addressed separately based on the GPIB address number when passing the adapter into the instrument construction. ::
+To instead use a Prologix GPIB device connected on :code:`/dev/ttyUSB0` (proper permissions are needed in Linux, see :class:`PrologixAdapter <pymeasure.adapters.PrologixAdapter>`), the adapter is constructed in a similar way.
+The Prologix adapter can be shared by many instruments.
+Therefore, new :class:`PrologixAdapter <pymeasure.adapters.PrologixAdapter>` instances with different GPIB addresses can be generated from an already existing instance. ::
 
     from pymeasure.adapters import PrologixAdapter
 
-    adapter = PrologixAdapter('/dev/ttyUSB0')
-    sourcemeter = Keithley2400(adapter.gpib(4))
+    adapter = PrologixAdapter('ASRL/dev/ttyUSB0::INSTR', address=7)
```

Comment:
It's unfortunate that `PyVISA` addressing name requires this `ASRL` string prefixed which creates incompatibility with the past. `PyVISA` documentation is not clear (at least to me) about this naming scheme. I have checked also in Windows and apparently the form `ASRLCOM5` is not valid. I think only option in Windows is `ASRL5`, even though is not guaranteed that `COMn` will be translated to `ASRLn`. A bit of a mess, but none of your fault.

Reply:
To make it more complicated, some backends (NI) also accept "COM5", while others (pyvisa) require "ASRL5".

Reply:
> To make it more complicated, some backends (NI) also accept "COM5", while others (pyvisa) require "ASRL5".

Indeed that's the case. This inconsistency is tracked in the second part of this report: https://github.com/pyvisa/pyvisa-py/issues/230. I don't know if this something from the VISA standard that's not implemented in pyvisa-py, or if the NI library does something "beyond the specification" when accepting `COM5`.